### PR TITLE
Add more Object methods

### PR DESCRIPTION
**Pull request by lucacasonato**
Specifically Object::get_own_property_descriptor
and Object::get_property_attributes

### CODE REVIEW

File: src/object.rs [hunk 1]
```diff
@@ -755,6 +767,45 @@ impl Object {
...
+        &*key,
+        &mut out,
+      )
+    };
+    out.into()
+  }
+
+  /// Implements Object.getOwnPropertyDescriptor(O, P), see
+  /// https://tc39.es/ecma262/#sec-object.getownpropertydescriptor.
+  pub fn get_own_property_descriptor<'s>(
```
Comment (bartlomieju) [inline src/object.rs:794]: Any chance you could test a code path that returns `None`?

File: src/object.rs [hunk 2]
```diff
@@ -755,6 +767,45 @@ impl Object {
     }
     .into()
   }
+
+  /// Gets the property attributes of a property which can be [None] or any
+  /// combination of [PropertyAttribute::READ_ONLY],
+  /// [PropertyAttribute::DONT_ENUM] and [PropertyAttribute::DONT_DELETE].
+  /// Returns [None] when the property doesn't exist.
+  pub fn get_property_attributes(
```
Comment (bartlomieju) [inline src/object.rs:775]: Ditto

Comment (lucacasonato) [reply src/object.rs:776]: Done.

File: src/object.rs (hunk 1 above)
Comment (lucacasonato) [reply src/object.rs:795]: V8 seems to never return an Empty MaybeLocal here. They also don't test this in their test_api.cc.

**Review by bartlomieju — APPROVED** [review]
LGTM

**Pull request merged**